accessibility-checker-engine/src/v2/checker/accessibility/util/legacy.ts

```diff
@@ -3178,22 +3178,27 @@ export class RPTUtil {
         var ancestors = [];
         let walkNode : Element = ruleContext;
         while (walkNode) {
-            if (walkNode.nodeType === 1)
+            if (walkNode.nodeType === 1) 
                 ancestors.push(walkNode);
             walkNode = DOMUtil.parentElement(walkNode);
         }
-
+        
         var retVal = {
             "hasGradient": false,
             "hasBGImage": false,
             "fg": null,
             "bg": null
         };
-        var compStyleColor = win.getComputedStyle(ruleContext).color;
+        
+        // start
+        var cStyle = win.getComputedStyle(ruleContext);
+        var compStyleColor = cStyle.color;
+        var compStyleBgColor = cStyle.backgroundColor;
         if (!compStyleColor)
             compStyleColor = "black";
         var fg = RPTUtil.Color(compStyleColor);
-
+        var bg = RPTUtil.Color(compStyleBgColor);
```

Comment:
Doesn't seem this additional line is used?

Reply:
removed